inferno-framework/inferno-framework.github.io
# Create Content on Community Section

# Summary
Four new pages have been added to the community section of the documentation site.

# Testing Guidance

- Run `bundle exec jekyll serve`
- Navigate to `localhost:4000/index_new.html`, then go to the `Community` page.
- Confirm that the content in the community section is accurate
